lib/main.js

```diff
@@ -87,23 +82,29 @@ ava.test.before(async () => {
   const resinConfiguration = await resinio.getDeviceOSConfiguration(
     placeholder.uuid, placeholder.deviceApiKey, _.assign({
       version: options.resinOSVersion
-    }, options.configuration))
-
-  console.log(`Injecting resin.io configuration into ${imagePath}`)
-  await resinos.injectResinConfiguration(imagePath, resinConfiguration)
+    }, options.configuration)
+  )
+
+  context.os = new Resinos(path.join(options.tmpdir, 'resin.img'), {
```

Comment:
I guess that with the modifications I proposed above, we can pass `options.tmpdir` to the constructor, and let the actual class figure out the final location